spec/bookingsync/api/client/bookings_spec.rb

```diff
@@ -117,4 +117,42 @@
         body: { bookings: [{ cancelation_reason: "payment_failed" }] }.to_json
     end
   end
+
+  describe ".add_bookings_fee_booking", :vcr do
+    let(:created_booking_id) {
+      find_resource("#{@casette_base_path}_create_booking/creates_a_booking.yml", "bookings")[:id]
+    }
+
+    it "adds bookings_fee" do
+      client.add_bookings_fee(created_booking_id, price: 100, times_booked: 2, name_en: "foobar")
+      assert_requested :patch, bs_url("bookings/#{created_booking_id}/add_bookings_fee"),
+        body: { bookings_fees: [{ price: 100, times_booked: 2, name_en: "foobar" }] }.to_json
+    end
+
+    it "returns with bookings_fee" do
+      VCR.use_cassette("BookingSync_API_Client_Bookings/_add_bookings_fee/adds_bookings_fee") do
+        booking = client.add_bookings_fee(created_booking_id, price: 100, times_booked: 2, name_en: "foobar")
+        expect(booking).to be_kind_of(BookingSync::API::Resource)
+        expect(booking.bookings_fees).to eq(
+          [{:id=>194398, :booking_id=>840043, :rentals_fee_id=>nil, :times_booked=>2, :price=>"100.0",
```

Comment:
Would best best to format this returned hash a little better, using new hash syntax and clear line break between each bookings fee